man/r2agent.1

```diff
@@ -1,7 +1,7 @@
 .Dd Jul 28, 2016
 .Dt R2AGENT 1
 .Sh NAME
-.Nm R2AGENT
+.Nm r2agent
```

Comment:
this is wrong. check `man ls` or `man vim` and you'll see those words are in always in uppercase